tools/integration-tests/tester/framework/drngnetwork.go

```diff
@@ -49,15 +48,6 @@ func (n *DRNGNetwork) CreatePeer(c GoShimmerConfig, publicKey ed25519.PublicKey)
 		config.FPCTotalRoundsFinalization = 10
 	}
 
-	// create wallet
```

Comment:
so this wallet is never created for the faucet? for what was the c.Faucet option then actually?

Reply:
Yes, in this drng scenario, nodes don't send funds around, so neither the faucet nor a wallet is needed. 
`c.Faucet` means we need one guy in the network to be the faucet :smile: